patches/0001-Instrument-Mender-client-for-coverage-analysis.patch

```diff
@@ -23,8 +23,8 @@ index 98e138a..ecb5730 100644
  
  	"github.com/mendersoftware/mender/app"
  	"github.com/mendersoftware/mender/cli"
-@@ -32,6 +34,21 @@ func init() {
- 	signal.Notify(cli.SignalHandlerChan, syscall.SIGUSR1, syscall.SIGUSR2, syscall.SIGTERM)
+@@ -35,6 +37,21 @@ func init() {
+ 	signal.Notify(termSignalChan, syscall.SIGTERM)
```

Comment:
I think you are reading it wrong. The new patch contains only the `signal.Notify(termSignalChan, syscall.SIGTERM)` line, which is still there. The long line doesn't exist in the patch anymore because it is outside the diff distance.

Reply:
Oh, you are right :see_no_evil: